### PR TITLE
Improve makedoc.g readability using triple-quotes

### PR DESCRIPTION
**Pull request by fingolfin**

### CODE REVIEW

File: makedoc.g [hunk 1]
```diff
@@ -7,18 +7,22 @@ if fail = LoadPackage("AutoDoc", "2019.04.10") then
...
+AutoDoc(rec(
+            scaffold := rec(
+                gapdoc_latex_options := rec(
+                LateExtraPreamble := """
+                    \usepackage{amsmath}
+                    \usepackage[T1]{fontenc}
+                    \usepackage{tikz}
+                    \usetikzlibrary{shapes,arrows,matrix}
+                    \usepackage{faktor}
+                    \RecustomVerbatimEnvironment{Verbatim}{BVerbatim}{}
```
Comment (zickgraf) [inline makedoc.g]:
```suggestion
```

This line should only appear in `makedoc_with_overfull_box_warnings.g`

File: makedoc.g [hunk 2]
```diff
@@ -7,18 +7,22 @@ if fail = LoadPackage("AutoDoc", "2019.04.10") then
...
-            scaffold := rec( gapdoc_latex_options := rec( 
-                             LateExtraPreamble := "\\usepackage{amsmath}\\usepackage[T1]{fontenc}\n\\usepackage{tikz}\n\\usetikzlibrary{shapes,arrows,matrix}\n\\usepackage{faktor}" 
-                                                        ),
-                             entities := [ "GAP4", "CAP" ],
-                             ),
-            
+AutoDoc(rec(
+            scaffold := rec(
+                gapdoc_latex_options := rec(
+                LateExtraPreamble := """
```
Comment (zickgraf) [inline makedoc.g]: Should this be indented one level more? (I'm not sure what the intedentation standard for `rec`s with only a single entry is.)

Comment (fingolfin) [reply makedoc.g]: There aren't really any such standards, and this file never followed any, but I agree this should be formatted differently. I'll update it.

**Comment by fingolfin** [comment]
Updated.

**Comment by zickgraf** [comment]
Perfect, thanks for this PR!

**Pull request merged**